crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -144,7 +144,11 @@ async fn app_validation_workflow_inner(
     let jh = tokio::spawn(async move {
         while let Some(op) = iter.next().await {
             // Send the result to task that will commit to the database.
-            if tx.send(op).await.is_err() {
+            if tx
```

Comment:
Again, don't risk blocking forever here. Not sure what the timeout should be, but never finishing is too long